Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/Delete.ts

```diff
@@ -43,6 +44,10 @@ export async function deleteDocumentById(
 
       if (referenceResult.rows == null) {
         await client.query('ROLLBACK');
+        const errorMessage = `deleteDocumentById: Error determining documents referenced by ${id},
+        a null result set was returned`;
```

Comment:
I like lines that aren't very wide, but not at the expensive of having a newline show up in a log file 😆 
```suggestion
        const errorMessage = `deleteDocumentById: Error determining documents referenced by ${id}, a null result set was returned`;
```